src/attached_probe.cpp

```diff
@@ -960,11 +964,39 @@ void AttachedProbe::attach_kprobe(bool safe_mode)
     return;
   }
 
+  // Construct a string containing "module:function."
+  // Also log a warning or throw an error if the module doesn't exist,
+  // before attempting to attach.
+  // Note that we do not pass vmlinux, if it is specified.
+  std::string funcname = probe_.attach_point;
+  const std::string &modname = probe_.path;
+  if ((modname.length() > 0) && modname != "vmlinux")
+  {
+    if (!is_loaded_module(modname))
+    {
+      if (probe_.orig_name != probe_.name)
+      {
+        // Wildcard usage just gets a warning
+        LOG(WARNING) << "specified module " << modname << " in probe "
+                     << probe_.name << " is not loaded.";
```

Comment:
Could we prepare the message into a string and then send it either to `LOG(WARNING)` or `LOG(ERROR)`? That way, when we want to change the message, we just do it in one place.